cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -82,7 +82,7 @@ func (m *Manager) Deploy(ctx context.Context) (*DeployResult, error) {
 		m.console.StopSpinner(ctx, "Didn't find new changes.", input.StepSkipped)
 	}
 
-	if err := m.UpdateEnvironment(ctx, m.env, deployResult.Deployment.Outputs); err != nil {
+	if err := m.UpdateEnvironment(ctx, deployResult.Deployment.Outputs); err != nil {
 		return nil, fmt.Errorf("updating environment with deployment outputs: %w", err)
 	}
```

Comment:
`env` function param no longer required since it should always be leveraging the internal `env`.